### ISSUE TITLE
Make a IBC FAQ

### ISSUE THREAD

**Issue by okwme** [cosmos/ibc#324]
I'm encountering the same questions more often. Would be good to collect them in the same place.

**Comment by okwme** [comment]
Unclear where this should live?
I've been interested in trying to use Stack Overflow for more Cosmos related questions to avoid repetition (and because it has really good SEO). I don't know if it makes sense to performative ask those questions and then answer them ourselves, maybe we can link an FAQ to stack overflow questions though.

Maybe this should live on something like the ibcprotocol.org website that has been discussed?

I could begin collecting questions here in the mean time...

Questions:
 * __What happens to all of the established IBC channels if a chain forks?:__
 *  This depends on the light client algorithm. Tendermint light clients, at the moment, will halt completely if a fork is detected (since it looks like equivocation) - if the fork doesn't use any sort of replay protection (e.g. change the chain ID). If one fork keeps the chain ID and the other picks a new one, the one which keeps it would be followed by the light client. If both forks change the chain ID (or validator set), they would both need new light clients.
* __What happens after the unbonding period passes without an IBC packet to renew the channel? Are the escrowed tokens un-recoverable without intervention?__
* By default, the tokens are un-recoverable. Governance intervention could alter the light client associated with the channel (there is no way to automate this that is safe). That said, it's always possible to construct light clients with different validation rules or to add the ability for a government proposal to reset the light client to a trusted header if it was previously valid and used, and if it was frozen due to the unbonding period.

**Comment by okwme** [comment]
* __Does Blockchain A need to know the address of a trustworthy node for Blockchain B in order to send IBC packets?__
 * Blockchain A will know of the existence of Blockchain B after a kind of handshake takes place. This handshake is facilitated by a relayer. It is the responsibility of the relayer to access an available node of the corresponding blockchain to begin the handshake. The blockchains themselves need not know about nodes, just be able to access the transactions that are relayed between them.

(is this correct?)

**Comment by cwgoes** [comment]
> * **Does Blockchain A need to know the address of a trustworthy node for Blockchain B in order to send IBC packets?**
> * Blockchain A will know of the existence of Blockchain B after a kind of handshake takes place. This handshake is facilitated by a relayer. It is the responsibility of the relayer to access an available node of the corresponding blockchain to begin the handshake. The blockchains themselves need not know about nodes, just be able to access the transactions that are relayed between them.
> 
> (is this correct?)

Yes - the relayers relay transactions / packets to the chains; the chains do not directly contact other chains themselves (indeed, they could not do so deterministically).

**Comment by Michelle-L** [comment]
@okwme and @cwgoes I would love to help with this needed. Maybe we could scour for questions popping up and then try and bucket them into certain "topics" ? Let me know Billy how I can support.

**Comment by cwgoes** [comment]
Let's turn this into a Markdown file in this repo; then additions can be PRs to that file.

**Comment by cwgoes** [comment]
I've added a FAQ file with the above questions & answers [here](https://github.com/cosmos/ics/blob/master/ibc/6_IBC_FAQ.md). PRs welcome.

**Comment by okwme** [comment]
on the weekend no less!
thank you <3
